app/src/main/java/com/github/se/eventradar/ui/hosting/Hosting.kt

```diff
@@ -0,0 +1,281 @@
+            start.linkTo(parent.start)
+            end.linkTo(parent.end)
+          })
+    }
+    // To Do: Floating Action Point Button. when button clicked, it should turn into mapView +
+    // button logo changes to List logo
+    Row(
+        modifier =
+            Modifier.constrainAs(ref = buttons) {
+                  bottom.linkTo(bottomNav.top, margin = 10.dp)
+                  centerHorizontallyTo(parent)
+                }
+                .testTag("floatingActionButtons")
+                .fillMaxWidth()
+                .padding(16.dp),
+        horizontalArrangement = Arrangement.Absolute.Left,
+        verticalAlignment = Alignment.CenterVertically) {
+          ExtendedFloatingActionButton(
+              text = {
+                Text(
+                    text = "Create New Event",
+                    style =
+                        TextStyle(
+                            fontSize = 16.sp,
+                            lineHeight = 17.sp,
+                            fontFamily = FontFamily(Font(R.font.roboto)),
+                            fontWeight = FontWeight(500),
+                            textAlign = TextAlign.Center,
+                            letterSpacing = 0.25.sp,
+                        ))
+              },
+              icon = { Icon(Icons.Filled.Add, "Floating action button.") },
+              onClick = { /*TODO*/},
+              modifier = Modifier.fillMaxWidth(0.8f).testTag("createEventButton"),
+              containerColor = MaterialTheme.colorScheme.secondaryContainer,
+              contentColor = MaterialTheme.colorScheme.onSecondaryContainer)
+          Spacer(modifier = Modifier.width(16.dp))
+          FloatingActionButton( // to be replaced with mapview later.
+              onClick = { /*TODO*/},
+              modifier = Modifier.fillMaxWidth().testTag("switchViewButton"),
+              containerColor = MaterialTheme.colorScheme.secondaryContainer,
+              contentColor = MaterialTheme.colorScheme.onSecondaryContainer,
+          ) {
+            Icon(
+                painter = painterResource(id = R.drawable.map),
+                contentDescription = "register to event button",
+                modifier = Modifier.size(32.dp).padding(5.dp),
+                tint = MaterialTheme.colorScheme.primary,
+            )
+          }
+        }
+
+    BottomNavigationMenu(
+        onTabSelected = { tab -> navigationActions.navigateTo(tab) },
+        tabList = TOP_LEVEL_DESTINATIONS,
+        selectedItem = TOP_LEVEL_DESTINATIONS[3],
+        modifier =
+            Modifier.testTag("bottomNavMenu").constrainAs(bottomNav) {
+              bottom.linkTo(parent.bottom)
+              start.linkTo(parent.start)
+              end.linkTo(parent.end)
+            })
+  }
+}
+
+@Composable
+fun EventList(events: List<Event>, modifier: Modifier = Modifier) {
+  LazyColumn(modifier = modifier) { items(events) { event -> EventCard(event) } }
+}
```

Comment:
Ahhh... Nevermind then. So the `EventList` is imported from Home, but also redefined (same for EventCard). There's a """huge""" problem of code duplication in your project guys! Pay attention to what your teammates write, and use the features already implemented instead of copy-pasting the same code in a diffeerent file!